gitea/user_app.go

```diff
@@ -22,6 +22,7 @@ func BasicAuthEncode(user, pass string) string {
 type AccessToken struct {
 	Name string `json:"name"`
 	Sha1 string `json:"sha1"`
+	ID   int64  `json:"id"`
```

Comment:
All other structs `ID` is a first field so for readability it would be better to move it up also in this struct

Reply:
Thanks. Updated.